app/views/marc_fields/_linked_author_index.html.erb

```diff
@@ -0,0 +1,7 @@
+<% if linked_author.present? %>
+  <% linked_author.values.each do |author| %>
+    <%= link_to(author[:link].strip, catalog_index_path(q: '"' + author[:search] + '"', search_field: 'search_author')) %>
+    <%= author[:post_text] if author[:post_text] %>
+    <br/>
```

Comment:
i can move the `<li>` tags into this view partial. i ported this over from the field marc partial and it was to break up the venacular.